python_modules/dagster-graphql/dagster_graphql_tests/graphql/test_permissions.py

```diff
@@ -73,6 +77,20 @@ def test_check_permission_permission_does_not_exist(fake_graphene_info):
         mutation.mutate(fake_graphene_info)
 
 
+def test_assert_permission_has_permission(fake_graphene_info):
+    assert assert_permission(fake_graphene_info, "fake_permission")
```

Comment:
in line with above, this reads a bit funny to me (since the assert will never return False)

Reply:
The assertion was here to make sure that the "truthiness" of a successful `assert_permission` call functions as desired, so the "if guard" model can work. If we end up removing the `return True` / not having the "if guard" model, there's no need for this.